roles/add_kernel_parameters/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2024 RTE
+# Copyright (C) 2024 Savoir-faire Linux, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
```

Comment:
You should add a check to know if the kernel parameters is already present. 
Like it is done here : https://github.com/seapath/ansible/blob/debiancentos/roles/yocto/kernel_params/tasks/main.yaml#L9